site/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask.ipynb

```diff
@@ -0,0 +1,2084 @@
+       "</svg>\n",
+       "</td>\n",
+       "</tr>\n",
+       "</table></pre></li></ul></div></li><li class='xr-section-item'><input id='section-452bf50e-2022-4867-8289-79b6f638542f' class='xr-section-summary-in' type='checkbox'  ><label for='section-452bf50e-2022-4867-8289-79b6f638542f' class='xr-section-summary' >Attributes: <span>(11)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'><dt><span>title :</span></dt><dd>/workspace/jhamman/processed/R1002RBRxaaa01a/lnd/temp/R1002RBRxaaa01a.vic.ha.1979-09-01.nc</dd><dt><span>institution :</span></dt><dd>U.W.</dd><dt><span>source :</span></dt><dd>RACM R1002RBRxaaa01a</dd><dt><span>output_frequency :</span></dt><dd>daily</dd><dt><span>output_mode :</span></dt><dd>averaged</dd><dt><span>convention :</span></dt><dd>CF-1.4</dd><dt><span>references :</span></dt><dd>Based on the initial model of Liang et al., 1994, JGR, 99, 14,415- 14,429.</dd><dt><span>comment :</span></dt><dd>Output from the Variable Infiltration Capacity (VIC) model.</dd><dt><span>nco_openmp_thread_number :</span></dt><dd>1</dd><dt><span>NCO :</span></dt><dd>&quot;4.6.0&quot;</dd><dt><span>history :</span></dt><dd>Tue Dec 27 14:15:22 2016: ncatted -a dimensions,,d,, rasm.nc rasm.nc\n",
+       "Tue Dec 27 13:38:40 2016: ncks -3 rasm.nc rasm.nc\n",
+       "history deleted for brevity</dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset>\n",
+       "Dimensions:  (time: 36, x: 275, y: 205)\n",
+       "Coordinates:\n",
+       "    yc       (y, x) float64 dask.array<chunksize=(205, 275), meta=np.ndarray>\n",
+       "    xc       (y, x) float64 dask.array<chunksize=(205, 275), meta=np.ndarray>\n",
+       "  * time     (time) object 1980-09-16 12:00:00 ... 1983-08-17 00:00:00\n",
+       "Dimensions without coordinates: x, y\n",
+       "Data variables:\n",
+       "    Tair     (time, y, x) float64 dask.array<chunksize=(12, 205, 275), meta=np.ndarray>\n",
+       "Attributes:\n",
+       "    title:                     /workspace/jhamman/processed/R1002RBRxaaa01a/l...\n",
+       "    institution:               U.W.\n",
+       "    source:                    RACM R1002RBRxaaa01a\n",
+       "    output_frequency:          daily\n",
+       "    output_mode:               averaged\n",
+       "    convention:                CF-1.4\n",
+       "    references:                Based on the initial model of Liang et al., 19...\n",
+       "    comment:                   Output from the Variable Infiltration Capacity...\n",
+       "    nco_openmp_thread_number:  1\n",
+       "    NCO:                       \"4.6.0\"\n",
+       "    history:                   Tue Dec 27 14:15:22 2016: ncatted -a dimension..."
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_ds = xr.open_mfdataset(paths, combine='by_coords')\n",
+    "new_ds"
```

Comment:
Can we do something more rigorous than just visual inspection that these two datasets are the same?  Can we do some equality checking?

Reply:
Done!